ocaml/xapi/extauth_plugin_ADwinbind.ml

```diff
@@ -340,7 +341,12 @@ module Ldap = struct
           @ attrs
         in
         let stdout =
-          Helpers.call_script ~env ~log_output !Xapi_globs.net_cmd args
+          match timeout with
```

Comment:
This can be simplified: 
```
          Helpers.call_script ~env ~log_output ?timeout !Xapi_globs.net_cmd args 
```

See https://v2.ocaml.org/manual/lablexamples.html

```
It may also be useful to relay an optional argument from a function call to another. This can be done by prefixing the
 applied argument with ?. This question mark disables the wrapping of optional argument in an option type.

# let test2 ?x ?y () = test ?x ?y () ();;
val test2 : ?x:int -> ?y:int -> unit -> int * int * int = <fun>
# test2 ?x:None;;
- : ?y:int -> unit -> int * int * int = <fun>
```